src/NLog/Internal/ObjectPools/ReusableBuilderCreator.cs

```diff
@@ -41,7 +41,13 @@ namespace NLog.Internal
     internal sealed class ReusableBuilderCreator : ReusableObjectCreator<StringBuilder>
     {
         public ReusableBuilderCreator()
-            : base(128, (cap) => new StringBuilder(cap), (sb) => { sb.ClearBuilder(); })
+            : base(128, (cap) => new StringBuilder(cap), (capacity, sb) =>
```

Comment:
Can `capacity` be renamed to `maxCapacity` ?

Reply:
i'm not sure that it's maxCapacity. Because at the moment the capacity can be as large as desired until we reset this value

Reply:
for stringBuilder maxCapacity means that you really can't exceed this size
![image](https://github.com/user-attachments/assets/a7486b53-0f25-448d-9c52-92ad53f51182)

Reply:
I only requested the input-parameter to be renamed. The StringBuilder should still be allowed to grow (to ensure log-events are not thrown away), but when StringBuilder has grown beyond the maxCapacity, then Capacity should be reset.

Reply:
fixed